dapp/src/utils/time.ts

```diff
@@ -94,3 +94,8 @@ export const getExpirationTimestamp = (duration: number, startDate?: Date) => {
 
   return dateToUnixTimestamp(expirationDate)
 }
+
+export const isWithinPeriod = (
+  timestamp: number,
+  periodInMilliseconds: number,
+) => Date.now() - timestamp <= periodInMilliseconds && timestamp <= Date.now()
```

Comment:
I do not understand it 😆 `Date.now()` is in miliseconds. I understand the `timestamp`  as a UNIX timestamp (seconds) so the condition `timestamp <= Date.now()` is always true.

Reply:
We use this function in `AcreRankCard`. However, this component is not used anywhere. Let's back to this topic when we focus on the leaderboard and ranks. 

https://github.com/thesis/acre/pull/704/commits/0bec1554c0fd578f2fe1610b6046b2df8e020270